### ISSUE TITLE
Not able create or write in external sdcard (removable memory card) ?

### ISSUE THREAD

**Issue by bjambu20** [passy/Android-DirectoryChooser#96]
Hi. I am integrated your code to application and its works fine with device storage but i am unable to use removable sd card(memory card) but path is selectable. not able to create new directory with the choosen path from this app.

**Comment by Edward-Yung** [comment]
I encounter this problem too. :(

**Comment by bjambu20** [comment]
Have you solved this issue @Edward-Yung . Please any guide me to solve this problem.

**Comment by Edward-Yung** [comment]
I finally found out it is the google's problem. It changes the spec such that app cannot write to secondary external storage from android 5.0. And I solved it by detecting if it is the removable sd card to hide all those files by modifying the original source code.

**Comment by DennyWeinberg** [comment]
You are right. I implemented a workaround for my app by using this source code: http://stackoverflow.com/questions/26744842/how-to-use-the-new-sd-card-access-api-presented-for-lollipop

But I don't want to do this for this library. Can you explain us how you modified the code in order to be able to select the sd card folder by simply hiding its content?

**Comment by 88bb** [comment]
You may try this app: https://github.com/Edward-Yung/Mu-Retargeter-1.0 to see if the implemented file chooser is working as expected as you want. If so, you can take a look at the source code and use my modified library. :)

**Comment by 88bb** [comment]
You may try this app: https://github.com/Edward-Yung/Mu-Retargeter-1.0 to see if the implemented file chooser is working as expected as you want. If so, you can take a look at the source code and use his modified library. :)

**Comment by apacha** [comment]
Or take a look at this app: https://github.com/arpitkh96/AmazeFileManager to see how they implemented write-access to external SD-card by using [MediaStoreHack](https://github.com/arpitkh96/AmazeFileManager/blob/master/src/main/java/com/amaze/filemanager/filesystem/MediaStoreHack.java)
